vms/subscriber.go

```diff
@@ -20,8 +20,8 @@ type Subscriber interface {
 	// ProcessFromHeight processes events from {height} to the latest block
 	ProcessFromHeight(height *big.Int) error
 
-	// UpdateLatestSeenBlock retrieves the latest block from the chain and updates the database
-	UpdateLatestSeenBlock() error
+	// UpdateLatestProcessedBlock retrieves the latest block from the chain and updates the database
+	UpdateLatestProcessedBlock() error
```

Comment:
I'm more in favor of omitting `Initial` from the function name, since the interface imposes no requirement that this function be called at any particular time in the object lifecycle. I agree it could be more descriptive as to what it's actually updating the database value _to_. I've renamed it `SetProcessedBlockHeightToLatest`; let me know if you think another name more accurately reflects the behavior.

Reply:
Nice, yeah totally agree. Good call 🙏